frontend/src/structuralComponents/Form3Rows.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import "./css/Form3Rows.css";
+
+type ParentParams = {
+    children: React.ReactNode
+}
+
+function Form3Rows(props: ParentParams) {
+    return (
+        <div className={"form-3-rows"}>
+            {props.children}
+        </div>
+    );
+}
```

Comment:
whats the difference to FieldLabelGroup? Both wrap the children in <div>

Reply:
They have different css classes. FieldLabelGroup groups a formfield and its label together